docs/sandbox/apps/sarotate.md

```diff
@@ -18,10 +18,51 @@ sb install sandbox-sarotate
 
 ```
 
-### 2. URL
+### 2. Usage
+
+- After installing the role, add your SA path to the settings file in `/opt/sandbox/settings.yml`
+
+- Example
+
+  ``` { .yaml}
+  sarotate: 
+  remotes: ["google"]
+  sa_path: "/opt/sa/all" # Change this as needed
+  sleeptime: # Optional. Default is 300
+  rc_port: # Default is localhost:5572
+  rc_user: # Optional. Default is blank
+  rc_pass: # Optional. Default is blank
+  apprise: # Optional. Add an apprise url if it suits you
+
+  ```
+
+**remotes:** if you have a different union in your rclone.conf (typically located @ `/home/user/.config/rclone/`) you will want to supply that name here. The saltbox default is google.
 
-- To access sarotate, visit `https://sarotate._yourdomain.com_`
+**sa_path:** supply the path to your service accounts. Saltbox default is `/opt/sa/all`
 
-### 3. Usage
+Heres a short example of the `config.yaml` in `/opt/sarotate/`.
```

Comment:
Since the role populates a config on its own I don't think this is needed and may be confusing - we can point to the upstream docs for those that want to customize the config

Reply:
Removed section from page.